stdlib/Cargo.toml

```diff
@@ -63,6 +63,8 @@ ahash = "0.7.6"
 libz-sys = { version = "1.1.5", optional = true }
 num_enum = "0.5.7"
 ascii = "1.0.0"
+once_cell = "1.13.0"
+mac_address = "1.1.3"
```

Comment:
Adding this line instead of `dependencies.uuid` section
will be more easy to read

```
uuid = { version = "1.1.2", features = ["v1", "fast-rng", "macro-diagnostics"] }
```

Reply:
`mac_address` needs more cfg guard for more OS.
See `mmap` case. By referring CI failure, we (at least) need to exclude iOS and android.